Packs/AppID_PAC/Playbooks/playbook-Select_App-ID_Payload_Results__Push_to_Firewall_and_Close_Incident.yml

```diff
@@ -0,0 +1,769 @@
+contentitemexportablefields:
+  contentitemfields:
+    propagationLabels:
+    - all
+id: Select App-ID Payload Results, Push to Firewall and Close Incident
+inputs: []
+name: Select App-ID Payload Results, Push to Firewall and Close Incident
+outputs: []
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "2"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 2b79622d-76bd-4356-8d47-e9ba6d63a21a
+      iscommand: false
+      name: ""
+      version: -1
+      description: ''
+    taskid: 2b79622d-76bd-4356-8d47-e9ba6d63a21a
+    timertriggers: []
+    type: start
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 50
+        }
+      }
+  "1":
```

Comment:
Please add description key to task number: 1,3,4,7,11
can see examples: https://xsoar.pan.dev/docs/playbooks/playbook-conventions